docs/handling-text-input.md

```diff
@@ -3,7 +3,7 @@ id: handling-text-input
 title: Handling Text Input
 ---
 
-[`TextInput`](textinput#content) is a [Core Component](intro-react-native-components) that allows the user to enter text. It has an `onChangeText` prop that takes a function to be called every time the text changed, and an `onSubmitEditing` prop that takes a function to be called when the text is submitted.
+[`TextInput`](textinput#content) is a [Core Component](intro-react-native-components) that allows the user to enter text. It has an `onChangeText` prop that takes a function to be called every time the text changed.
```

Comment:
I believe this change is not exhaustive. Here other occurrences of `onSubmitEditing`

```
keyboard.md
38:        onSubmitEditing={Keyboard.dismiss}

improvingux.md
66:        onSubmitEditing={() => emailInput.current.focus()}
79:        onSubmitEditing={submit}
160:        onSubmitEditing={() => emailInput.current?.focus()}
173:        onSubmitEditing={submit}
262:          onSubmitEditing={submit}
359:          onSubmitEditing={submit}

textinput.md
8:The most basic use case is to plop down a `TextInput` and subscribe to the `onChangeText` events to read the user input. There are also other events, such as `onSubmitEditing` and `onFocus` that can be subscribed to. A minimal example:
222:If `true`, the text field will blur when submitted. The default value is true for single-line fields and false for multiline fields. Note that for multiline fields, setting `blurOnSubmit` to `true` means that pressing return will blur the field and trigger the `onSubmitEditing` event instead of inserting a newline into the field.
658:### `onSubmitEditing`
```